test/test_models/hf-internal-testing/tiny-random-gpt2-fp16/genai_config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "model": {
+    "type": "gpt2",
+    "pad_token_id": 98,
+    "bos_token_id": 98,
+    "eos_token_id": 98,
+    "vocab_size": 1000,
+    "logits_type": "float16",
```

Comment:
Could we add the capability to infer the type from model?

Reply:
Yep, that'll be in the next PR. This unblocks people using the updated builder.py